content/07.pathogenesis.md

```diff
@@ -274,37 +274,35 @@ For example, because of the high level of sequence homology between SARS-CoV-2 a
 However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, which would be likely to prohibit effective cross-neutralization across different SARS species [@doi:10.1101/2020.03.21.990770].
 Consequently, proteomic analyses of SARS-CoV-1 might also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].
 
-Considering the paucity of omics-level big data sets for SARS-CoV-2 currently available, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for comparative bioinformatics research of SARS-CoV-2.
-Using such databases, the systems-level reconstruction of protein-protein interaction (PPI) networks will enable the generation of hypotheses about the mechanism of action of SARS-CoV-2 and suggest potential drug targets.
+Considering the paucity of omics-level big data sets for SARS-CoV-2 currently available, existing data hubs that contain information for other coronaviruses such as UniProt [@doi:10.1093/nar/gky1049], NCBI Genome Database [@url:https://www.ncbi.nlm.nih.gov/genome/], The Immune Epitope Database and Analysis Resource [@doi:10.1093/nar/gky1006], and The Virus Pathogen Resource [@doi:10.1093/nar/gkr859] will serve as useful resources for comparative bioinformatics research of SARS-CoV-2.
```

Comment:
Totally fine if you want to remove abbrev here. I'll just say there I know for virologists..these abbrev are very well known and common..though i guess this manuscript will be fore a general audience so that makes sense

Reply:
I am not sure how to handle this because I agree they are (and like should I change NCBI to National Center for Biotechnology Information? seems silly!) I guess I was hesitant because the mSystems guidelines specifically say don't give an acronym unless you use it 3 times!

Reply:
They'll find this when they do proofs 🤷  - i think we ship it :shipit: